### PR TITLE
Adding a workflow for creating a docker image

### PR DESCRIPTION
**Pull request by sicheste**
Adding a GitHub job for creating a and pushing a docker image if a tag is created.

### CODE REVIEW

File: .github/workflows/publish_docker.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Docker login
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```
Comment (andig) [inline .github/workflows/publish_docker.yml:23]:
```suggestion
          username: ${{ secrets.DOCKER_USER }}
          password: ${{ secrets.DOCKER_PASS }}
```

File: .github/workflows/publish_docker.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        uses: docker/setup-buildx-action@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: volkszaehler/mbmd
+
+      - name: Build and push
+        uses: docker/build-push-action@v5
```
Comment (andig) [inline .github/workflows/publish_docker.yml:35]:
```suggestion
        uses: docker/build-push-action@v6
```

Comment (andig) [reply .github/workflows/publish_docker.yml:35]: Do we need an additional

```
      - name: Setup Buildx
        uses: docker/setup-buildx-action@v3
```

?

Comment (sicheste) [reply .github/workflows/publish_docker.yml]: See lines 25 to 26. Buildx is already there.

**Comment by andig** [comment]
Great PR, thank you!

**Pull request merged**